src/classes/BDI_DataImport_TEST2.cls

```diff
@@ -1494,20 +1494,36 @@ private with sharing class BDI_DataImport_TEST2 {
         npe01__Contacts_and_Orgs_Settings__c contactSettingsForTests = UTIL_CustomSettingsFacade.getContactsSettingsForTests(
```

Comment:
I'd consider moving the Adv logic to a separate test, as the method name doesn't match what you're testing in that context